### ISSUE TITLE
Porting Coin3D to WEB with WASM

### ISSUE THREAD

**Issue by kaankoroglu** [coin3d/coin#514]
Hi, 

I am looking for a open inventor viewer in WEB environment and searched the internet but could not find any alternative. So my question is, is that possible to compile Coin3D libraries with the Emscripten so that I can use it on the WEB environment.

If it is not the correct place to talk about, I can move my question to the discussions part.

Thanks in advance.

**Comment by VolkerEnderlein** [comment]
Yes, I think moving the question to the `Discussions` area would be a good step to get more devs and users involved.
